docs/guides/getting-started-orchestrate-human-tasks.md

```diff
@@ -0,0 +1,74 @@
+---
+id: getting-started-orchestrate-human-tasks
+title: Getting started with Human Task Orchestration
+sidebar_label: Getting started with Human Task Orchestration
+description: "Efficiently allocate work through user tasks."
+keywords: [human tasks, orchestration, getting-started]
+---
+
+## Getting started with user task orchestration
+
+### Purpose
+
+Using [Camunda Cloud](./getting-started/create-camunda-cloud-account.md), you can orchestrate human tasks by assigning them to users. Then, users can enter the necessary data to drive the business process.
+
+When a process instance arrives at such a user task, a new job similar to a service task is created. The process instance stops at this point and waits until the job is completed. Applications like [Tasklist](../components/tasklist/introduction.md) can be used by humans to complete these tasks.
+
+While it may originally seem like the goal of automating a process is to remove humans entirely, efficiently allocating work through user tasks can be even more beneficial.
+
+In this guide, we’ll step through one way to create an automated process utilizing user tasks – all entirely executable in the browser.
+
+### Prerequisites
+
+- Ensure you have a valid [Camunda Cloud account](./getting-started/create-camunda-cloud-account.md), or sign up if you still need one.
+- (Optional) Install [Camunda Desktop Modeler](../components/modeler/desktop-modeler/install-the-modeler.md).
+
+### Create an automated process with user tasks
+
+To create an automated process with user tasks, take the following steps:
+
+#### Develop your automated process with user tasks
+
+1. Log in to your Camunda Cloud account.
+2. To create a BPMN diagram, navigate to Web Modeler via the **Modeler** tab, and click **New project**.
+3. Name your project and select **New > BPMN Diagram > + Create blank**.
+4. Give your model a descriptive name, and then give your model a descriptive id within the **General** tab inside the properties panel on the right side of the screen.
```

Comment:
Can you include an example of a descriptive name and id? 

Also, I noticed for Web Modeler, the name of the diagram (as in the file) does NOT update when you update the name and id on the properties panel. I didn't call this out in the MO guide updates, but can you check with @ultraschuppi and team if this is expected behavior? For completeness, we should really encourage naming the diagram file.